scripts/variantstore/wdl/HailFromWdl.wdl

```diff
@@ -0,0 +1,250 @@
+version 1.0
+# Largely "borrowing" from Lee's work
+# https://github.com/broadinstitute/aou-ancestry/blob/a57bbab3ccee4d06317fecb8ca109424bca373b7/script/wdl/hail_in_wdl/filter_VDS_and_shard_by_contig.wdl
+
+#
+# Given a VDS and a bed file, render a VCF (sharded by chromosome).
+#   All bed files referenced in this WDL are UCSC bed files (as opposed to PLINK bed files).
+#
+# This has not been tested on any reference other than hg38.
+# Inputs:
+#
+#         ## ANALYSIS PARAMETERS
+#        #  ie, parameters that go to the Hail python code (submission_script below)
+#        String vds_url
+#
+#        # Genomic region for the output VCFs to cover
+#        String bed_url
+#
+#        # VCF Header that will be used in the output
+#        String vcf_header_url
+#
+#        # Contigs of interest.  If a contig is present in the bed file, but not in this list, the contig will be ignored.
+#        #   In other words, this is a contig level intersection with the bed file.
+#        #     This list of contigs that must be present in the reference.  Each contig will be processed separately (shard)
+#        # This list should be ordered.  Eg, ["chr21", "chr22"]
+#        Array[String] contigs
+#
+#        # String used in construction of output filename
+#        #  Cannot contain any special characters, ie, characters must be alphanumeric or "_"
```

Comment:
Underscore or dash?? You use dashes below when defining prefix

Reply:
Yeah the comment is wrong as Rori discovered the hard way, must be a dash.